training/src/main/resources/demo.bash

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# rename training-all.jar to aerosolve-generic-pipeline-current.jar
+# set driver-memory and executor-memory for your job.
+# ./demo.bash demo_train.conf GenericDebugExamples
+# if you have additional java options (options pass to conf) i.e. ds=2016-01-01
+# ./demo.bash demo_train.conf GenericDebugExamples ds=2016-01-01
+
+/usr/bin/abb-spark-15-submit \
```

Comment:
Is this for public audience? If so, should probably change to `$SPARK_HOME/spark-submit` instead.